frontend/src/app/commonComponents/Header.scss

```diff
@@ -1,5 +1,6 @@
 .usa-nav-container.prime-header {
   //   padding: 0;
```

Comment:
can we remove the commented padding?

Reply:
Good eye! 🕵️ removed in [e7ec589](https://github.com/CDCgov/prime-simplereport/pull/4759/commits/e7ec589c9eaaaa56a85025e3bd89d9b7a4a5f8e4).